pyteal/ast/txn.py

```diff
@@ -37,7 +37,7 @@ class TxnField(Enum):
     sender = (0, "Sender", TealType.bytes, False, 2)
     fee = (1, "Fee", TealType.uint64, False, 2)
     first_valid = (2, "FirstValid", TealType.uint64, False, 2)
```

Comment:
@jdtzmn Minor:  The addition looks correct.  To complete the effort, can you extend https://pyteal.readthedocs.io/en/stable/accessing_transaction_field.html#common-fields to reflect the new field?

For better or worse, the user guide's field definition listing isn't auto-generated.

Reply:
@jdtzmn Thanks for the change in https://github.com/algorand/pyteal/pull/424/commits/1095b848e9ad108e2c74f91bb2a47e29d1fb6adc - resolving.